test/mock-results.js

```diff
@@ -15,17 +15,30 @@ module.exports = function(stats, options) {
         testsuite: [
           {
             _attr: {
-              name: "Foo Bar module",
+              name: "Root Suite",
```

Comment:
The old stubs never emitted the root suite, but in reality mocha will always have one and we'll always output it.